### DISCUSSION TITLE
How to advertise our job to the community?

### DISCUSSION THREAD

**Discussion by ronalducci** [community/community#114886] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I am interested in finding an experienced developer through GH for our new startup idea which has attracted funding. How to start advertising and how much does it cost?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @ronalducci , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
